vhdbuilder/release-notes/AKSCBLMarinerV2/gen1/latest.txt

```diff
@@ -586,24 +597,12 @@ wireless-regdb-2022.08.12-1.cm2.noarch
 gpg-pubkey-3135ce90-5e6fda74
 gpg-pubkey-be1229cf-5631588c
 inotify-tools-3.22.1.0-1.cm2.x86_64
-mariner-repos-shared-2.0-9.cm2.noarch
-hyperv-daemons-license-5.15.138.1-1.cm2.noarch
 ca-certificates-tools-2.0.0-14.cm2.noarch
 ca-certificates-shared-2.0.0-14.cm2.noarch
-hypervfcopyd-5.15.138.1-1.cm2.x86_64
-hypervkvpd-5.15.138.1-1.cm2.x86_64
-hypervvssd-5.15.138.1-1.cm2.x86_64
-ncurses-libs-6.4-2.cm2.x86_64
-ncurses-6.4-2.cm2.x86_64
-ncurses-term-6.4-2.cm2.x86_64
-hyperv-daemons-5.15.138.1-1.cm2.x86_64
+openssl-libs-1.1.1k-28.cm2.x86_64
```

Comment:
openssl 1.1.1k-27.cm2.x86__64 1.1.1k-28.cm2 has a vulnerability, this week's VHD has bumped it to .28